script.js

```diff
@@ -161,7 +161,11 @@ function handleData([sheetsText, statesGeoJson]) {
     }, new Map());
 
   const localitiesData = rows.filter(
-    row => row.admin_scale !== "State" && row.lat !== null && row.lon !== null
+    row => row.admin_scale !== "State" && row.lat !== null && row.lon !== null && row.admin_scale !=="County"
```

Comment:
@goldfarb you could simplify this by just doing:
```js
  const localitiesData = rows.filter(
    row => row.admin_scale === "City" && row.lat !== null && row.lon !== null
  );
```
no need to use the two `!==` for "County" and "State" when you can use a single `===` here